src/System.Threading.Thread/src/System/Threading/Thread.cs

```diff
@@ -315,6 +415,7 @@ private static int ToTimeoutMilliseconds(TimeSpan timeout)
         [CLSCompliant(false)]
         public static void VolatileWrite(ref UIntPtr address, UIntPtr value) => Volatile.Write(ref address, value);
 
+#if !NETNATIVE
```

Comment:
This is the local slot implementation, unless I missed something, these don't depend on any runtime support. It seems like it shouldn't be necessary to disable them.